CHANGELOG.rst

```diff
@@ -2,19 +2,35 @@
 Changelog
 =========
 
+`2023.7 <https://github.com/python/python-docs-theme/releases/tag/v2023.7>`_
+----------------------------------------------------------------------------
+
+- Do not append asset hashes on Sphinx 7.1+ (#137)
+   Contributed by Pradyun Gedam
+- Use correct block, enable built-in slash keypress (#141)
+   Contributed by Mike Fiedler
+- Sphinx 6.2 fix: add ``nav.contents`` where that ``div.topic`` is used (#138)
+   Contributed by Hugo van Kemenade
+- Dark mode: fix contrast for C++ specific styling (#133)
```

Comment:
```suggestion
- Dark mode: fix contrast for ``C++`` specific styling (#133)
```

Reply:
Thanks! I added all suggestions except this as we don't need code literals for proper nouns.